code/typescript/pipelines-workshop/lib/cdk-workshop-stack.ts

```diff
@@ -14,7 +13,7 @@ export class CdkWorkshopStack extends cdk.Stack {
 
     const hello = new lambda.Function(this, 'HelloHandler', {
       runtime: lambda.Runtime.NODEJS_14_X,
-      code: lambda.Code.asset(path.resolve(__dirname, '../lambda')),
+      code: lambda.Code.fromAsset('lambda'),
```

Comment:
Deprecated API. Also CI and local testing passes without using `path.resolve()` so I removed it  for sake of simplicity.